src/main/java/mdlaf/MaterialLookAndFeel.java

```diff
@@ -728,9 +726,6 @@ public void setTheme(MaterialTheme theme) {
 
     public static int getFocusAcceleratorKeyMask() {
         Toolkit tk = Toolkit.getDefaultToolkit();
-        if (tk instanceof SunToolkit) {
```

Comment:
Why you removed this code?

Reply:
Because SunToolkit isn't available in Java 9+. We need to find a cross-version alternative, but I think just returning ActionEvent.ALT_MASK is fine for now?

Reply:
Mh I don't know I will test your last version of the code and I will return back here.

Please, before you can fix [this problem](https://github.com/vincenzopalazzo/material-ui-swing/pull/119#discussion_r392619968)?